tests/PHPStan/Analyser/data/early-termination-phpdoc.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace EarlyTermination;
+
+use PHPStan\TrinaryLogic;
+use function PHPStan\Analyser\assertVariableCertainty;
+
+function(): void {
+	$pre = 1;
+	Foo::doBarPhpDoc();
+	$a = 1;
+	assertVariableCertainty(TrinaryLogic::createYes(), $pre);
```

Comment:
I'm not even sure if these are executed. They might not be. Have you seen them fail?

The typical test here should be this instead:

```php
if (rand(0, 1)) {
    $a = 1;
} else {
    Foo::doBarPhpDoc();
}

assertVariableCertainty(TrinaryLogic::createYes(), $a);
```